libopeniscsiusr/session.c

```diff
@@ -291,7 +291,7 @@ int iscsi_sessions_get(struct iscsi_context *ctx,
 	}
 	/* reset session count and sessions array length to what we were able to read from sysfs */
 	*session_count = j;
-	*sessions = reallocarray(*sessions, *session_count, sizeof(struct iscsi_session *));
+	*sessions = realloc(*sessions, *session_count * sizeof(struct iscsi_session *));
```

Comment:
Please add a brief comment to the effect that we are using realloc() instead of reallocarray(), and why, and we assume the multiplication of "count" times "size" won't overflow -- something reallocarray() checks for us and realloc() does not.

Reply:
After that, I'll be glad to merge this.